packages/lib/constants.ts

```diff
@@ -61,6 +61,7 @@ export const MAIL_FROM = env.MAIL_FROM;
 export const NEXTAUTH_SECRET = env.NEXTAUTH_SECRET;
 export const NEXTAUTH_URL = env.NEXTAUTH_URL;
 export const ITEMS_PER_PAGE = 50;
+export const OPEN_TEXT_RESPONSES_PER_PAGE = 10;
```

Comment:
Not sure, but can we reduce this default size to maybe 5. Since, 10 is still a large number, and makes it look still long. This is optional, you can discuss it and figure out if it makes sense to reduce this to 5.